### ISSUE TITLE
Fox model not working

### ISSUE THREAD

**Issue by LoganDark** [ray-cast/ray-mmd#115]
This fox model is not working, the texture is all white!

![image](https://user-images.githubusercontent.com/4723091/83883114-e9ed5700-a6f7-11ea-9a54-05bf55ea41d8.png)

But everything else works, like that Umbreon!

[Fox (Red).zip](https://github.com/ray-cast/ray-mmd/files/4736440/Fox.Red.zip)

What's going on?

![image](https://user-images.githubusercontent.com/4723091/83883584-84e63100-a6f8-11ea-9865-6ae38741f281.png)

**Comment by Fau13F** [comment]
try putting another main.fx like ex_noalpha to see if it does something ?

**Comment by LoganDark** [comment]
Nope, none of them worked

**Comment by Fau13F** [comment]
and in all the tabs the model has his effects assigned ? (like material_2.0.fx, PSSM1..)

**Comment by LoganDark** [comment]
Yep and changing the material had no effect

Everything is in perfect working order, completely identical between the fox and the Umbreon, except the fox doesn't work!

This is probably a technical problem that only the maintainers/coders of ray-mmd can fix...

**Comment by Fau13F** [comment]
yes sorry i don't know what to do more for you, and since the creator seems to have left the project, nothing can be done :/

**Comment by tERB00** [comment]
![1](https://user-images.githubusercontent.com/17136839/104106050-bbee7780-52c3-11eb-99a3-60233793f656.jpg)
I have no issues with the model. Looks like you have incorrectly set your lighting. Because this model have opaque 0.95 you have that weird effect on transparency.

**Comment by LoganDark** [comment]
> ![1](https://user-images.githubusercontent.com/17136839/104106050-bbee7780-52c3-11eb-99a3-60233793f656.jpg)
> I have no issues with the model. Looks like you have set your lighting incorrectly. Because this model have opaque 0.95 you have that weird effect on transparency.

What's your setup? Also what's your GPU?

**Comment by tERB00** [comment]
> What's your setup? Also what's your GPU?
> If I'm doing something wrong, please tell me what I can do to fix it :(
It's AMD Radeon.
There are a ton of step-by-step tutorials on how to use ray-mmd on youtube. Go watch them.

**Comment by LoganDark** [comment]
> There are a ton of step-by-step tutorials on how to use ray-mmd on youtube. Go watch them.

You seem to be mistaking me for a new and confused user who doesn't know how to use ray-mmd. I know perfectly well how to use it, aside for the fact that I don't know what to change to fix this one, specific issue. Everything else I've been able to do perfectly fine.

I'm not asking for a step by step tutorial from you. I'm just asking for you to tell me why the model turns white for me, other than "you're doing it wrong, follow a youtube tutorial".

**Comment by tERB00** [comment]
> > There are a ton of step-by-step tutorials on how to use ray-mmd on youtube. Go watch them.
> 
> You seem to be mistaking me for a new and confused user who doesn't know how to use ray-mmd. I know perfectly well how to use it, aside for the fact that I don't know what to change to fix this one, specific issue. Everything else I've been able to do perfectly fine.
> 
> I'm not asking for a step by step tutorial from you. I'm just asking for you to tell me why the model turns white for me, other than "you're doing it wrong, follow a youtube tutorial".

I did not made any extra steps to make you model work on my machine. All I did is loaded your model and applied main.fx on to it.

**Comment by LoganDark** [comment]
> > > There are a ton of step-by-step tutorials on how to use ray-mmd on youtube. Go watch them.
> > 
> > 
> > You seem to be mistaking me for a new and confused user who doesn't know how to use ray-mmd. I know perfectly well how to use it, aside for the fact that I don't know what to change to fix this one, specific issue. Everything else I've been able to do perfectly fine.
> > I'm not asking for a step by step tutorial from you. I'm just asking for you to tell me why the model turns white for me, other than "you're doing it wrong, follow a youtube tutorial".
> 
> I did not made any extra steps to make you model work on my machine. All I did is loaded your model and applied main.fx on to it.

Alright. I will try that out the next time I boot into Windows, I'll tell you how it goes.

**Comment by tERB00** [comment]
> Alright. I will try that out the next time I boot into Windows, I'll tell you how it goes. I don't remember if I did the same thing or not, I'll record my attempt so you can tell me if I make a mistake.

If you won't succeed doing that my advice is to make fresh clean install of MMD. Just download latest versions of MMD, MME and ray-mmd and put them in a separate folder.

**Comment by tERB00** [comment]
I bet you did not set up EnvLightMap properly...

**Comment by LoganDark** [comment]
> I bet you did not set up EnvLightMap properly...

I will check out, but please try not to be so rude.
